Cargo.toml

```diff
@@ -20,12 +20,14 @@ base64 = "0.21.0"
 env_logger = "0.10.0"
 fastrand = "1.9.0"
 git-version = "0.3.5"
+janus_messages = { git = "https://github.com/divviup/janus.git" }
```

Comment:
We probably should add versioning of some sort to this eventually so changes on main don't break builds

Reply:
This is published at https://crates.io/crates/janus_messages and there haven't been big changes in a while, we should use that.

Reply:
resolved in 78e8cef41ad48b282a765bdaec479cfebdb6cf47